pages/patterns/create-a-profile/pronouns.md

```diff
@@ -112,7 +112,7 @@ Provide a text entry field that supports a rich array of special characters and
 {:.site-component-section-title}
 ## References
 - Ask users for gender or sex. (n.d.) Retrieved on July 19, 2022, from [https://design-system.service.gov.uk/patterns/gender-or-sex/](https://design-system.service.gov.uk/patterns/gender-or-sex/)
-- Asking about gender in online forms. (September 18, 2015) Retrieved on July 19, 2022, from [http://www.practicemakesprogress.org/blog/2015/9/18/asking-about-gender-on-online-forms](http://www.practicemakesprogress.org/blog/2015/9/18/asking-about-gender-on-online-forms)
+- Asking about gender in online forms. (September 18, 2015) Retrieved on July 19, 2022, from http://www.practicemakesprogress.org/blog/2015/9/18/asking-about-gender-on-online-forms
```

Comment:
Removed for now in 2835c17, but follow-up captured in #2520.

Reply:
I also added a link to an archive.org capture on this one. I can add that to the SOP in the Engineering Guide